.github/workflows/secureSignal-cd.yaml

```diff
@@ -11,10 +12,17 @@ env:
 
 jobs:
   verify:
-    if: "!startsWith(github.event.head_commit.message, 'bump:')"
-    uses: ./.github/workflows/verify-file-changed.yml
-    with:
-      files: src/secureSignal.ts
+    runs-on: ubuntu-latest
+    outputs:
+      isAnyFileModified: ${{ steps.verify.outputs.any_modified }}
```

Comment:
Maybe this can be `is_any_file_modified` to match the formatting of other input/output vars?

Reply:
Updated